in_command.go

```diff
@@ -34,12 +34,9 @@ func (c *InCommand) Run(destDir string, request InRequest) (InResponse, error) {
 	var foundRelease *github.RepositoryRelease
 	var commitSHA string
 
-	if request.Version.Tag != "" {
-		foundRelease, err = c.github.GetReleaseByTag(request.Version.Tag)
-	} else {
-		id, _ := strconv.Atoi(request.Version.ID)
-		foundRelease, err = c.github.GetRelease(id)
-	}
+	id, _ := strconv.Atoi(request.Version.ID)
+	foundRelease, err = c.github.GetRelease(id)
+
```

Comment:
Minor nit: whitespace is usually omitted before `if err != nil {` check.

Reply:
Fixed.